### ISSUE TITLE
Syntelencephaly

### ISSUE THREAD

**Issue by patt1morgan** [obophenotype/human-phenotype-ontology#7032]
**Preferred term label:**  Syntelencephaly

**Synonyms** Middle Interhemispheric Variant of Holoprosencephaly, MIH, midline interhemispheric fusion, MIF

**Definition (free text, please give PubMed ID)** Syntelencephaly is a rare malformation that consists of an abnormal midline connection of the cerebral hemispheres in the posterior frontal and parietal regions, with interhemispheric separation of the basal forebrain, anterior frontal lobes, and occipital regions. PMID: 11827888

**Comment   this condition differs from HPE in that the midline "fusion" in syntelencephaly is complete but segmental, while the structural brain anomalies seen in the HPE spectrum progress smoothly in severity in a posterior to anterior "fusion." PMID: 8989462
**Parent term (use hpo.jax.org/app)**   Holoprosencephaly HP:0001360

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)** Mutations involving the ZIC2 gene, located on human chromosome 13q32, has been postulated in the pathogenesis of HPE and syntelencephaly. PMID: 26557166
**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@yuval-yaron, @suz13.  Thank you for agreeing to work with us to advance the perinatal section of the HPO. We would appreciate it if you could review this new term request and approve it as it is, or help us improve it. Any suggestions welcome!

**Comment by Suz13** [comment]
Agree relevant prenatal term. Agree parent term holoprosencephaly. 
Prenatal phenotype reported in this abstract https://obgyn.onlinelibrary.wiley.com/doi/full/10.1002/uog.15498 and prenatal case series in PMID:33455493.
ZIC2 also associated in PMID:29442327

**Comment by yuval-yaron** [comment]
Agree relevant term for prenatal diagnosis.

**Comment by pnrobinson** [comment]
added term (add annotations)

**Comment by pnrobinson** [comment]
added term, will add annotations.
